.github/workflows/deploy-lambda-endpoints.yml

```
@@ -32,30 +38,84 @@ jobs:
+          docker buildx build \
+          --build-arg LAMBDA_TASK=${{ matrix.folder }} \
+          --cache-from type=local,src=/tmp/.buildx-cache \
+          --cache-to type=local,dest=/tmp/.buildx-cache \
+          --load \
+          -t ${{ matrix.folder }}:${{ github.sha }} .
+    
+
+      - name: Save Docker image as artifact
+        run: |
+          docker save ${{ matrix.folder }}:${{ github.sha }} -o /tmp/${{ matrix.folder }}.tar
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)